client/landing/stepper/declarative-flow/internals/steps-repository/unified-plans/index.tsx

```diff
@@ -164,7 +162,7 @@ export default function PlansStepAdaptor( props: StepProps ) {
 			onPlanIntervalUpdate={ onPlanIntervalUpdate }
 			intervalType={ planInterval }
 			wrapperProps={ {
-				hideBack: isMobile,
+				hideBack: false,
```

Comment:
It's also fine to make `wrapperProps.hideBack` optional, since UnifiedPlansStep already accepts undefined for the `hideBack` prop.